recipes/cpplogging/all/conanfile.py

```diff
@@ -0,0 +1,100 @@
+from conans import CMake
```

Comment:
Given this is a new recipe I would be really appreciated if you used the newer build helpers

This is deprecated and will be removed soon...  you can see the template for a good example https://github.com/conan-io/conan-center-index/blob/3bf8c6b296e145c13781e8ee7e9bca497d077cc7/docs/package_templates/cmake_package/all/conanfile.py#L7

Reply:
Thanks for the template, I'll look at it.
I left that one line because I struggled a bit when I tried to remove it and could not make the recipe to work.